src/content/getting-started.mdx

```diff
@@ -51,9 +51,11 @@ $ scoop install okteto
 
 Alternatively, you can directly download the binary [from GitHub](https://github.com/okteto/okteto/releases) or build it directly from the source code.
 
+> It is important that Okteto CLI is in your PATH to simplify the usage of the [Kubernetes credentials generated by Okteto](cloud/credentials.mdx). Kubeconfig file generated uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the cluster credentials via Okteto and it is required that Okteto is in your PATH and it is connected via `okteto context use http://okteto.example.com` with your instance (check the following section to know how to configure it).
```

Comment:
Since we already explained that in the reference, I think it's also repetitive (and confusing) to have it here, specially since we are not using kubeconfig on the getting started. 

```suggestion
```